deploy/cloudformation/manifest-pipeline-pipeline.yml

```diff
@@ -0,0 +1,723 @@
+  - Creates a CodePipeline that has a specific CodeBuild associated,
+  which creates a changed set from the package and deploys it to test and production.
+  - Requires the iiif-service pipeline to be built.
+
+Parameters:
+  ProdStackName:
+    Type: String
+    Description: The name of the CloudFormation stack to use when creating the production resources
+    Default: "mellon-manifest-pipeline"
+  TestStackName:
+    Type: String
+    Description: The name of the CloudFormation stack to use when creating the test resources
+    Default: "mellon-manifest-pipeline-test"
+  Receivers:
+    Type: String
+    Description: An e-mail address to send the monitoring notifications
+  ConfigurationRepoName:
+    Type: String
+    Description: The GitHub repo for the cloudfromation blueprints
+    Default: mellon-blueprints
+  ConfigurationRepoBranchName:
+    Type: String
+    Description: The GitHub repo branch the codepipeline should checkout to run blueprints from
+    Default: master
+  ManifestPipelineRepoName:
+    Type: String
+    Description: The GitHub repo name
+    Default: mellon-manifest-pipeline
+  ManifestPipelineRepoBranch:
+    Type: String
+    Description: The GitHub repo branch code pipelines should watch for changes on
+    Default: master
+  GitHubUser:
+    Type: String
+    Description: GitHub UserName. This username must have access to the GitHubToken.
+    Default: ndlib
+  GitHubToken:
+    NoEcho: true
+    Type: String
+    Description: "Secret. OAuthToken with access to Repo. Long string of characters and digits. Go to https://github.com/settings/tokens"
+  ImageServiceTestStackName:
+    Type: String
+    Default: "mellon-image-service-test"
+    Description: The name of the test IIIF image service stack
+  ImageServiceProdStackName:
+    Type: String
+    Default: "mellon-image-service"
+    Description: The name of the production IIIF image service stack
+  DataBrokerStackName:
+    Type: String
+    Default: "mellon-data-broker"
+    Description: The name of the shared data broker stack
+
+Resources:
+  CodeBuildTrustRole:
+    Type: 'AWS::IAM::Role'
+    Properties:
+      AssumeRolePolicyDocument:
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - codebuild.amazonaws.com
+            Action: 'sts:AssumeRole'
+      Path: /
+  CodeBuildRolePolicy:
+    Type: 'AWS::IAM::Policy'
+    Properties:
+      PolicyName: CodeBuildRolePolicy
+      PolicyDocument:
+        Statement:
+          - Effect: Allow
+            Action:
+              - 'logs:CreateLogGroup'
+              - 'logs:CreateLogStream'
+              - 'logs:PutLogEvents'
+            Resource:
+              - '*'
+          - Effect: Allow
+            Resource:
+              - '*'
+            Action:
+              - 's3:*'
+          - Effect: Allow
+            Resource:
+              - '*'
+            Action:
+              - 'kms:GenerateDataKey*'
```

Comment:
Is there a specific reason why we are generating a new key for this service? Probably more for @jonhartzler

Reply:
I did not put that in that I know of.   If it was there it was copied from the example or @jgondron added it.  As far as I know, I don't currently have anything that I am encrypting.

Reply:
I don't think this is needed, but I've realized I haven't audited this Policy at all. So, I'll take a look at the CodeBuildRolePolicy overall.

Reply:
@ialford This should be cleaned up, please take another look

Reply:
Looks good. 💯